src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -527,160 +384,102 @@
 short-summary: List deployments at resource group.
 examples:
   - name: List deployments at resource group.
-    text: az deployment group list -g testrg
```

Comment:
@zhoxing-ms , please note that _for clarity_, abbreviated reference names have been replaced with full parameter names.  I've also attempted to standardize on a `MyName` object naming convention.

Reply:
OK, nice work! 👍👍 Thanks so much for your contribution!